packages/chaire-lib-common/src/tasks/dataImport/data/osmGeojsonService.ts

```diff
@@ -298,13 +301,17 @@ const getGeojsonsFromRawData = (
                     features[i].type,
                     features[i].id
                 );
-                throw 'Missing OSM geojson';
+                if (options.continueOnMissingGeojson) {
```

Comment:
Maybe we should adapt the error message above? Should it says also "Or verify the OSM data"